osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -61,11 +63,13 @@ public void SetOsuLogo(OsuLogo logo)
 
         private readonly List<Button> buttonsTopLevel = new List<Button>();
         private readonly List<Button> buttonsPlay = new List<Button>();
+        private readonly Timer idleTimer = new Timer(idle_timer_timeout);
```

Comment:
The ``Timer`` class is never used in osu!lazer, try using the ``Scheduler`` field inside this class instead.
Save the ``ScheduledDelegate`` that is being returned by the ``AddDelayed`` method and cancel it if needed. That's what's being done normally. (e.g. https://github.com/ppy/osu/blob/master/osu.Game/Overlays/VolumeOverlay.cs#L123)